Sources/Plasma/Apps/plClient/plClient.cpp

```diff
@@ -148,6 +148,9 @@ You can contact Cyan Worlds, Inc. by email legal@cyan.com
 #ifdef PLASMA_PIPELINE_GL
     #include "pfGLPipeline/plGLPipeline.h"
 #endif
+#ifdef PLASMA_PIPELINE_METAL
+    #include "pfMetalPipeline/plMetalPipeline.h"
```

Comment:
This doesn't exist in this PR and should probably be removed (and the attempt to init the Metal Pipeline further down)

Reply:
Removed from the branch history.